Buildings/Resources/Scripts/Dymola/Examples/VAVReheat/Validation/TraceSubstance.mos

```diff
@@ -0,0 +1,3 @@
+﻿simulateModel("Buildings.Examples.VAVReheat.Validation.TraceSubstance", startTime=3500000, stopTime=4650000, numberOfIntervals=0, outputInterval=900, method="Cvode", tolerance=1e-06, resultFile="TraceSubstance");
+createPlot(id=1, position={0, 0, 1537, 628}, y={"VOut1.V_flow", "senSupFlo.V_flow"}, range={3500000.0, 4650000.0, -0.2, 2.6}, grid=true, colors={{28,108,200}, {238,46,47}}, displayUnits={"m3/s", "m3/s"});
```

Comment:
It would be good to also plot the CO2 concentration in the room. This helps making sure the control is setup correctly.

Reply:
I actually have not specified any actual CO2 concentrations.  I will do this.  Do you want me to add generation from people within the floor model?  I can't think of a way to do this just through an ``extends`` statement and without a change to the floor model to add generation calculations or inputs for them from a top-level model.

Reply:
Yes please. It would be good to add CO2 generated by people through an `extends` statement.

Reply:
Ok, actually I have an idea of what to do for this.

Reply:
@mwetter Please re-review, especially https://github.com/lbl-srg/modelica-buildings/pull/2486/commits/96a59b638856cb64c343a3536451699ca5450a6b, and let me know if what I implemented is a good way to go for this.  I can actually use this validation test implementation for the BOPTEST test case in https://github.com/ibpsa/project1-boptest/issues/273.